packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -16,104 +16,254 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
+    "cutButtonLabel": r"剪切",
+    "okButtonLabel": r"确定",
+    "pasteButtonLabel": r"粘贴",
+    "selectAllButtonLabel": r"全选",
+    "viewLicensesButtonLabel": r"查看许可证"
   },
   "en": const <String, String>{
     "openAppDrawerTooltip": r"Open navigation menu",
     "backButtonTooltip": r"Back",
     "closeButtonTooltip": r"Close",
     "nextMonthTooltip": r"Next month",
-    "previousMonthTooltip": r"Previous month"
+    "previousMonthTooltip": r"Previous month",
+    "licensesPageTitle": r"Licenses",
+    "cancelButtonLabel": r"CANCEL",
+    "closeButtonLabel": r"CLOSE",
+    "continueButtonLabel": r"CONTINUE",
+    "copyButtonLabel": r"COPY",
+    "cutButtonLabel": r"CUT",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"PASTE",
+    "selectAllButtonLabel": r"SELECT ALL",
+    "viewLicensesButtonLabel": r"VIEW LICENSES"
   },
   "de": const <String, String>{
     "openAppDrawerTooltip": r"Navigationsmenü öffnen",
     "backButtonTooltip": r"Zurück",
-    "closeButtonTooltip": r"Schließen ",
+    "closeButtonTooltip": r"Schließen",
     "nextMonthTooltip": r"Nächster Monat",
-    "previousMonthTooltip": r"Letzter Monat"
+    "previousMonthTooltip": r"Vorheriger Monat",
+    "licensesPageTitle": r"Lizenzen",
+    "cancelButtonLabel": r"ABBRECHEN",
+    "closeButtonLabel": r"SCHLIESSEN",
+    "continueButtonLabel": r"FORTSETZEN",
+    "copyButtonLabel": r"KOPIEREN",
+    "cutButtonLabel": r"AUSSCHNEIDEN",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"EINFÜGEN",
+    "selectAllButtonLabel": r"ALLES AUSWÄHLEN",
+    "viewLicensesButtonLabel": r"LIZENZEN ANZEIGEN"
   },
   "ja": const <String, String>{
     "openAppDrawerTooltip": r"ナビゲーションメニューを開く",
-    "backButtonTooltip": r"バック",
+    "backButtonTooltip": r"戻る",
     "closeButtonTooltip": r"閉じる",
-    "nextMonthTooltip": r"来月",
-    "previousMonthTooltip": r"前の月"
+    "nextMonthTooltip": r"前の月へ",
+    "previousMonthTooltip": r"前の月へ",
+    "licensesPageTitle": r"ライセンス",
+    "cancelButtonLabel": r"キャンセル",
+    "closeButtonLabel": r"閉じる",
+    "continueButtonLabel": r"次へ",
+    "copyButtonLabel": r"コピー",
+    "cutButtonLabel": r"カット",
+    "okButtonLabel": r"OK",
+    "pasteButtonLabel": r"貼付け",
+    "selectAllButtonLabel": r"全選択",
+    "viewLicensesButtonLabel": r"ライセンス表記"
   },
   "ru": const <String, String>{
     "openAppDrawerTooltip": r"Открыть меню навигации",
     "backButtonTooltip": r"назад",
     "closeButtonTooltip": r"Закрыть",
     "nextMonthTooltip": r"В следующем месяце",
-    "previousMonthTooltip": r"Предыдущий месяц"
+    "previousMonthTooltip": r"Предыдущий месяц",
+    "licensesPageTitle": r"Лицензии",
+    "cancelButtonLabel": r"ОТМЕНА",
+    "closeButtonLabel": r"ЗАКРЫТЬ",
+    "continueButtonLabel": r"ПРОДОЛЖАТЬ",
+    "copyButtonLabel": r"копия",
+    "cutButtonLabel": r"порез",
+    "okButtonLabel": r"ОК",
+    "pasteButtonLabel": r"Паст",
+    "selectAllButtonLabel": r"Выбрать все",
+    "viewLicensesButtonLabel": r"ПРОСМОТРЕТЬ ЛИЦЕНЗИИ"
```

Comment:
All upper case looks funny(inconsistent?)

Reply:
In theory all button labels should be uppercase, I guess the error is with the others.